plugins/json-logger/src/templates/logger.template.ts

```diff
@@ -0,0 +1,20 @@
+
+
+import { Injectable } from '@nestjs/common';
+import { PinoLogger } from 'nestjs-pino';
+
+@Injectable()
+export class LoggerService {
```

Comment:
@pnkjbee2 The service needs either to implement Nestjs LoggerService as per docs https://docs.nestjs.com/techniques/logger#custom-implementation as this implementation won't be usable in the final product. 
Have you tested the generated code with this plugin to see if all the logs are in the correct format?